apps/src/templates/teacherNavigation/teacher-navigation.module.scss

```diff
@@ -4,7 +4,7 @@
 .pageAndSidebar {
   display: flex;
   flex-direction: row;
-  min-height: 100%;
+  min-height: 100vh;
```

Comment:
It feels a little weird to me that the sidebar will take up the entire screen even if the content only takes up half. e.g. only a couple students, the majority of the page would be completely blank with just a sidebar. What do you think?

An alternative is to just make sure that there is a significant margin below the last option in the sidebar and between the footer and the page content.

Reply:
I think this is really a question for @dju90 and @MollyPeredo.  I suppose I am ok with it because the footer is pretty U. G. L. Y. :)

Reply:
Having the footer below the fold doesn't bother me. The animation when the footer shows up feels a little abrupt to me though—it looks like the margin at the bottom of the sidebar changes after the user scrolls down far enough to see the footer and then back up?

Wondering if Liam's alternative approach would keep the footer below the fold and maintain more consistent spacing between elements?

Reply:
Note: I just verified with Molly that the current version is fine and perhaps there we a video glitch causing the animation issue.